robot/Cumulus/resources/Basenpspobjects.py

```diff
@@ -0,0 +1,12 @@
+from robot.libraries.BuiltIn import BuiltIn
```

Comment:
I would like to recommend you rename this file to BaseObjects.py.  The NPSP is redundant, and only having the first letter capitalized is a bit strange.

This isn't a showstopper and it doesn't affect functionality, but this name kind of sticks out from the other files in this folder.